src/data/systemIntake.ts

```diff
@@ -63,7 +63,8 @@ export const initialSystemIntakeForm: SystemIntakeForm = {
   grtReviewEmailBody: '',
   decidedAt: null,
   businessCaseId: null,
-  submittedAt: null
+  submittedAt: null,
+  lcid: ''
```

Comment:
Added lifecycle id to the intake model because we need it for the banner. The backend isn't passing that back because it doesn't exist yet.